lms/djangoapps/branding/api.py

```diff
@@ -0,0 +1,135 @@
+from microsite_configuration import microsite
+
+log = logging.getLogger("edx.footer")
+
+
+def get_footer():
+    """ Get the footer links json
+
+    Returns:
+        Dict of footer links
+    """
+    site_name = microsite.get_value('SITE_NAME', settings.SITE_NAME)
+    context = dict()
+    context["copy_right"] = copy_right()
+    context["heading"] = heading()
+    context["logo_img"] = "{site_name}/sites/all/themes/atedx/images/edx-logo-footer.png".format(site_name=site_name)
+    context["social_links"] = social_links()
+    context["about_links"] = about_edx_link(site_name)
+
+    return JsonResponse({"footer": context}, 200)
+
+
+def copy_right():
+    """ Returns the copy rights text
+    """
+    data = _("(c) 2015 edX Inc. EdX, Open edX, and the edX and Open edX logos "
+             "are registered trademarks or trademarks of edX Inc.")
+
+    return data
+
+
+def heading():
+    """ Returns the heading text
+    """
+
+    data = ("EdX offers interactive online classes and MOOCs from the world's best universities," +
+            " colleges and organizations. Online courses from MITx, HarvardX, BerkeleyX, UTx and" +
+            " many other universities can be taken here. Topics include biology, business, chemistry," +
+            " computer science, economics, finance, electronics, engineering, food and nutrition," +
+            " history, humanities, law, literature, math, medicine, music, philosophy, physics, science," +
+            " statistics and more. EdX is a non-profit online initiative created by founding partners" +
+            " Harvard and MIT.")
+    return data
+
+
+def social_links():
+    """ Returns the list of social link of footer
+    """
+    return [
+        {
+            "provider": "facebook",
+            "title": _("Facebook"),
+            "url": "http://www.facebook.com/EdxOnline"
+        },
+        {
+            "provider": "twitter",
+            "title": _("Twitter"),
+            "url": "https://twitter.com/edXOnline"
+        },
+        {
+            "provider": "linkedin",
+            "title": _("LinkedIn"),
+            "url": "http://www.linkedin.com/company/edx"
+        },
+        {
+            "provider": "google",
+            "title": _("Google+"),
+            "url": "https://plus.google.com/+edXOnline"
+        },
+        {
+            "provider": "tumblr",
+            "title": _("Tumblr"),
+            "url": "http://edxstories.tumblr.com/"
+        },
+        {
+            "provider": "meetup",
+            "title": _("Meetup"),
+            "url": "http://www.meetup.com/edX-Global-Community"
+        },
+        {
+            "provider": "reddit",
+            "title": _("Reddit"),
+            "url": "http://www.reddit.com/r/edx"
+        },
+        {
+            "provider": "youtube",
+            "title": _("Youtube"),
+            "url": "https://www.youtube.com/user/edxonline?sub_confirmation=1"
+        },
+    ]
+
+
+def about_edx_link(site_name):
```

Comment:
Please get these from Django settings: https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L1767-L1783

You will need to use the `marketing_link()` function to retrieve the URLs:
https://github.com/edx/edx-platform/blob/820b0118f485f3bda36e06db0af1f0fad5ea28f6/common/djangoapps/edxmako/shortcuts.py#L28-L57